lib/api/bucketGetTagging.js

```diff
@@ -0,0 +1,105 @@
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const util = require('node:util');
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { checkExpectedBucketOwner } = require('./apiUtils/authorization/bucketOwner');
+const { pushMetric } = require('../utapi/utilities');
+const { errors, s3middleware } = require('arsenal');
+const escapeForXml = s3middleware.escapeForXml;
+
+//  Sample XML response:
+/*
+    <Tagging>
+        <TagSet>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
+        </TagSet>
+    </Tagging>
+*/
+
+/**
+ * Convert Versioning Configuration object of a bucket into xml format.
+ * @param {array} tags - set of bucket tag keys and values
+ * @return {string} - the converted xml string of the versioning configuration
+ */
```

Comment:
New comment, I would either extend array to include the specific data structure or create a typedef of what a tag is.
https://jsdoc.app/tags-type.html

```suggestion
/**
 * @typedef Tag
 * @type {object}
 * @property {string} Value - Value of our tag.
 * @property {string} Key - Key of the tag.
 */
/**
 * Convert Versioning Configuration object of a bucket into xml format.
 * @param {array.<Tag>} tags - set of bucket tags
 * @return {string} - the converted xml string of the versioning configuration
 */
```
Ideally if you added more details as this was mostly an example, but now we get full typing.

Reply:
done? https://github.com/scality/cloudserver/pull/4481/commits/96157ec37c9e4398618eb993ff7e5fb92c4e8141#diff-889d6969db5c5fda428f1b3f508e237cb4bc1419949c78bc0a146bcca7d9209bR25